.github/actions/pr-is-mergeable/action.yaml

```diff
@@ -0,0 +1,122 @@
+name: "PR Is Mergeable"
+description: "Checks whether the provided PR is approved and all status checks either succeeded or have been skipped"
+
+inputs:
+  pr-ref:
+    description: "The reference (i.e. either number or a branch) of the PR to check"
+    required: true
+  fail-on-unmergeable:
+    description: "Whether to fail the action if the PR is not mergeable"
+    required: false
+    default: "true"
+  repo:
+    description: "The repository of the PR"
+    required: false
+    default: ${{ github.repository }}
+  token:
+    description: "The GitHub access token (with PR read permissions) to access the PR"
+    required: false
+    default: ${{ github.token }}
+  excluded-check-runs:
+    description: "A JSON object, where the keys are the names of the workflows to exclude. The values are a list of step names to ignore. If the list is empty, all steps of the workflow will be ignored."
+    required: false
+    default: '{\"${{ github.workflow }}\": []}'
```

Comment:
(Question)

Doesn't this mean by default "all steps of workflow ${{ github.workflow }}" will be ignored? This doesn't have an effect on the action? Wouldn't default "{}" better fit, i.e. to not ignore anything?

Reply:
This whole file is unreadable. If it was approved for the Cloud SDK we can also approve it of the AI SDK.

Reply:
I thought you had higher standards for AI SDK 😁

Reply:
I removed the whole generation of pom files with plugins script. Maybe we get rid of this in 2030 😁